spikeextractors/extractors/neuroscopeextractors/neuroscopeextractors.py

```diff
@@ -29,16 +29,18 @@ class NeuroscopeRecordingExtractor(BinDatRecordingExtractor):
     Parameters
     ----------
     file_path : str
-        Path to the .dat file to be extracted
+        Path to the .dat file to be extracted.
+    gain : float
+        Numerical value that converts the native int dtype to microvolts.
     """
 
-    extractor_name = 'NeuroscopeRecordingExtractor'
+    extractor_name = "NeuroscopeRecordingExtractor"
     installed = HAVE_LXML
     is_writable = True
-    mode = 'file'
-    installation_mesg = 'Please install lxml to use this extractor!'  # error message when not installed
+    mode = "file"
+    installation_mesg = "Please install lxml to use this extractor!"
 
-    def __init__(self, file_path: PathType):
+    def __init__(self, file_path: PathType, gain: float):
```

Comment:
should have default value of 1.0

Reply:
I do have concerns about setting this as an optional with a default, since it could lead to the original issue we had. If it could truly be determined automatically, I would go with that but would it not be better to just force the user to think about it?

Reply:
* It's not needed for many common use-cases of a spike sorting pipeline, because it is not needed to do spike sorting
* It changes this API in a breaking way, which will certainly break any code that depends on it
* minor: It is different from the standard API for `RecordingExtractors`

Reply:
Granted, making it a requirement breaks any back-compatibility...

But then again, any previous instances of the use of this extractor wouldn't have recorded the traces in the correct units anyway... so should probably make people aware if they were using it in that capacity?

Reply:
OK, I'll make it optional then. Shall I throw a warning at least if it defaults?

Reply:
@alejoe91, what do you think? Do users of this package generally care about the conversion factor?

Reply:
I think this should be up to the user. I would leave it as an optional arg with no warning.